lib/python/src/bailo/core/agent.py

```diff
@@ -25,9 +29,12 @@ def __init__(
         """
         self.verify = verify
 
+        logger.debug("Base Agent initiated successfully.")
+
     def __request(self, method, *args, **kwargs):
         kwargs["verify"] = self.verify
 
+        logger.debug(f"{method} to {args[0]}")
```

Comment:
I don't think this is needed. Requests provide there own logging that does the same thing but with a bit more detail in terms of sessions and ports.

Reply:
Done